github/orgs_members.go

```diff
@@ -270,3 +270,33 @@ func (s *OrganizationsService) RemoveOrgMembership(user, org string) (*Response,
 
 	return s.client.Do(req, nil)
 }
+
+//ListPendingOrgInvitations returns a list of contains a role field which refers to the
+//Organization Invitation role and will be one of the following values:
+//direct_member, admin, billing_manager, hiring_manager, or reinstate.
+//If the invitee is not a GitHub member, the login field in the return hash will be null.
+//
+// GitHub API docs:
+// https://developer.github.com/v3/orgs/members/#list-pending-organization-invitations
```

Comment:
There's no benefit in putting the long URL on a new line. We're not hard-limited to the width of a [punch card](https://twitter.com/rob_pike/status/563801489190043648), it's fine to keep it on the same line.